src/pages/search/advanced_search/ScientificValueFilter.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, Checkbox, FormControlLabel } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { ResultParam } from '../../../api/searchApi';
+import { dataTestId } from '../../../utils/dataTestIds';
+
+enum ScientificValueLevels {
+  LevelZero = 'Unassigned,LevelZero',
+  LevelOne = 'LevelOne',
+  LevelTwo = 'LevelTwo',
+}
+
+export const ScientificValueFilter = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const scientificValueParam = searchParams.get(ResultParam.ScientificValue) ?? '';
+
+  const levelZeroSelected = scientificValueParam.includes(ScientificValueLevels.LevelZero);
+  const levelOneSelected = scientificValueParam.includes(ScientificValueLevels.LevelOne);
+  const levelTwoSelected = scientificValueParam.includes(ScientificValueLevels.LevelTwo);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, checked } = event.target;
+    const newSelectedScientificValues = { levelZeroSelected, levelOneSelected, levelTwoSelected, [name]: checked };
```

Comment:
Skjønner ikke helt denne jer 🤔 Det lages et objekt med de samme verdiene som vi allerede har tilgjengelig på linje 20-22, men vi bruker de ikke igjen.

```suggestion
    const newSelectedScientificValues = { [name]: checked };
```

Da står vi igjen med `const newSelectedScientificValues = { [name]: checked };` som sikker også kan forenkles litt